chirp/drivers/uvb5.py

```diff
@@ -674,6 +674,10 @@ def apply_limit(setting, obj):
         rs.set_apply_callback(apply_limit, self._memobj.limits)
         basic.append(rs)
 
+        # FIXME: FM Presets are currently broken
+        #        - Saved frequencies display as integers
+        #        - Can't key in a decimal point
+        #        - Even vallid entries (65, 66, 67, etc.) are not accepted
```

Comment:
I just merged the Float properties fix. Does that resolve this comment or is there more?